tests/test_metadata.py

```diff
@@ -0,0 +1,709 @@
+                    EC.visibility_of_element_located(
+                        (By.CSS_SELECTOR, '[data-test-file-list-item]')
+                    )
+                )
+                row = utils.find_row_by_name(files_page, new_file)
+
+                file_link = row.find_element_by_css_selector(
+                    '[data-test-file-list-link]'
+                )
+                file_link.click()
+                driver.switch_to.window(driver.window_handles[0])
+                file_id = metadata['data']['attributes']['path']
+                file_guid = osf_api.get_fake_file_guid(session, file_id)
+                return file_guid
+
+        except Exception:
+            logger.error('Server error occurred')
+            osf_api.delete_addon_files(
+                session, provider, current_browser=settings.DRIVER, guid=node_id
+            )
+
+
+@pytest.fixture()
+def file_metadata_page(driver, file_guid):
+    file_metadata_page = FilesMetadataPage(driver, guid=file_guid)
+    file_metadata_page.goto()
+    return file_metadata_page
+
+
+def get_funder_information(funder_name):
+    """This method is used to get the funder information for the
+    given funder name using api link"""
+    url = settings.FUNDER_INFO_URL
+    response = requests.get(url)
+    data = response.json()
+    for funder in data['included']:
+        if funder['type'] == 'index-card':
+            if (
+                funder['attributes']['resourceMetadata']['name'][0]['@value']
+                == funder_name
+            ):
+                award_title = funder['attributes']['resourceMetadata']['resourceType'][
+                    0
+                ]['@id']
+                award_uri = funder['attributes']['resourceIdentifier'][0]
+                award_number = funder['id']
+
+    return award_title, award_uri, award_number
+
+
+@pytest.mark.usefixtures('must_be_logged_in')
+@markers.smoke_test
+@markers.core_functionality
+class TestFilesMetadata:
+    @pytest.fixture()
+    def file_metadata_page_with_data(self, driver, file_metadata_page, fake):
+        title = fake.sentence(nb_words=1)
+        description = fake.sentence(nb_words=4)
+
+        file_metadata_page.files_metadata_edit_button.click()
+        file_metadata_page.edit_title.click()
+        title_input = driver.find_element_by_css_selector(
+            '[data-test-title-field] textarea'
+        )
+        title_input.send_keys(title)
+
+        description_input = driver.find_element_by_css_selector(
+            '[data-test-description-field] textarea'
+        )
+        description_input.send_keys(description)
+
+        # Select 'Book' from the resource type listbox
+        file_metadata_page.resource_type.click()
+        WebDriverWait(driver, 5).until(
+            EC.visibility_of_element_located(
+                (
+                    By.CSS_SELECTOR,
+                    '#ember-basic-dropdown-wormhole > div > ul > li>span',
+                )
+            )
```

Comment:
The above style might be an isort/black thing, but if not, I suggest the following:

```python
EC.visibility_of_element_located((
        By.CSS_SELECTOR,
        '#ember-basic-dropdown-wormhole > div > ul > li>span',
))
```